service/history/transferQueueActiveTaskExecutor.go

```diff
@@ -667,8 +669,8 @@ func (t *transferQueueActiveTaskExecutor) processStartChildExecution(
 		return err
 	}
 
-	// NOTE: do not access anything related mutable state after this lock release
-	// release the context lock since we no longer need mutable state builder and
+	// NOTE: do not access anything related mutable state after this Lock release
+	// release the context Lock since we no longer need mutable state builder and
```

Comment:
Same thing. There is a way in GoLand to search in comments only. I think you can easily revert these renames by searching for "Lock" and "Clear" case sensitive in comments only.